win32/devdoc/threadpool_win32.md

```diff
@@ -215,6 +215,8 @@ MOCKABLE_FUNCTION(, int, threadpool_timer_restart, TIMER_INSTANCE_HANDLE, timer,
 
 **SRS_THREADPOOL_WIN32_42_019: [** If `timer` is `NULL`, `threadpool_timer_restart` shall fail and return a non-zero value. **]**
 
+If timer is `DESTROYING`, `threadpool_timer_restart` shall fail and return a non-zero value.
```

Comment:
>If timer is [](http://example.com/codeflow?start=0&length=11)

I believe that this should read if timer state is DESTROYING ...

Reply:
fixed